### ISSUE TITLE
Workflow for remote drake_models + Bazel + binary Drake

### ISSUE THREAD

**Issue by jwnimmer-tri** [RobotLocomotion/drake#19094]
Some people use a pre-compiled version of Drake with Bazel as their build system, but do not declare `package://drake_models` as a repository external.

For example, the `drake_bazel_installed` workflow, or using `@rules_python//python:pip.bzl` with `pydrake`.

As it stands now after #19046, when running bazel tests in this case, the `$TEST_TMPDIR` sandboxing always wins, so all download caching is defeated.  This means tests could be doing a lot of downloading.

We should provide some way for those downstream tests to avoid repeated downloading.

**Comment by jwnimmer-tri** [comment]
I'm going to call this a duplicate of #19094.

**Comment by RussTedrake** [comment]
nit: your last line has the wrong issue number. (it's marked as a duplicate of itself)

**Comment by jwnimmer-tri** [comment]
Oops!  I meant #19823.
